wandb/sdk_py27/internal/handler.py

```diff
@@ -108,7 +146,7 @@ def handle_artifact(self, record):
     def handle_alert(self, record):
         self._dispatch_record(record)
 
-    def _save_summary(self, summary_dict, flush=False):
+    def _save_summary(self, summary_dict, flush = False):
```

Comment:
This seems to be a bug in `strip_type_annotations.sh`.

Reply:
True, filed a separate bug for that: https://wandb.atlassian.net/browse/WB-4435